app/controllers/api/v1/api_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Api::V1::ApiController < ActionController::API
+  rescue_from ActiveRecord::RecordInvalid, with: :record_invalid
+  rescue_from ActiveRecord::RecordNotFound, with: :not_found
+  rescue_from ActionController::ParameterMissing, with: :parameter_missing
+
+  private
+
+  def record_invalid(exception)
+    render json: exception.record.errors, status: :unprocessable_entity
+  end
+
+  def not_found(exception)
+    render json: { message: "#{model_name(exception.model)} não encontrado" }, status: :not_found
```

Comment:
```suggestion
    render json: { message: "#{exception.model.constantize.model_name.human} não encontrado" }, status: :not_found
```

Essa é meio maluca mas desde que você use o I18n você pode pegar o model em texto transformar numa constante e usar o I18n do nome do model como forma de tradução. Acho que ficaria legal, passar essa mensagem total pro I18n, talvez passando uma variável que vale o nome do model

https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models

Reply:
Eu estava pensando em fazer isso, mas não estava conseguindo. Não sabia que dava pra transformar o `exception.model ` em outra coisa. Vou tentar fazer desse jeito pra ver e se funcionar já colocar o I18n dos outros erros de retorno também